src/loader.js

```diff
@@ -1,70 +1,119 @@
 function alternatingCaseToObject(string) {
+    const config = string.match(/[A-Z]+[0-9a-z]+/g);
+
+    if (!config) return {};
+
     let o = {};
-    let match = string.match(/[A-Z]+[0-9a-z]+/g);
-    if (match) {
-        match.forEach(match => {
-            const [, key, value] = match.match(/([A-Z]+)([0-9a-z]+)/);
-            o[key.toLowerCase()] = value;
-        });
-    }
+
+    config.forEach(match => {
+        let [, key, value] = match.match(/([A-Z]+)([0-9a-z]+)/);
+        key = key.toLowerCase();
+
+        // Do some type guessing
+        if (parseFloat(value).toString() === value) {
+            value = parseFloat(value);
+        } else if (value === 'true' || value === 'yes') {
+            value = true;
+        } else if (value === 'false' || value === 'no') {
+            value = false;
+        }
+
+        if (o[key]) {
+            // Key exists so treat it as a list
+            if (!(o[key] instanceof Array)) {
+                o[key] = [o[key]];
+            }
+            o[key].push(value);
+        } else {
+            o[key] = value;
+        }
+    });
+
     return o;
 }
 
-function getGroups(group, idx, all) {
-    // While has next sibling and next sibling is valid (not a group and not null)
-    let allNodes = all.map(g => g.node);
-    let nextNode = group.node.nextSibling;
-    if (idx === 0 && !!nextNode) nextNode = nextNode.nextSibling;
-    let html = '';
-    while (nextNode && allNodes.indexOf(nextNode) < 0) {
-        if (nextNode.outerHTML) {
-            html += nextNode.outerHTML;
-        }
-        nextNode = nextNode.nextSibling;
+// Load any scrollyteller content from Odyssey
+let scrollytellers;
+function getScrollytellers() {
+    if (!scrollytellers) {
+        scrollytellers = window.__ODYSSEY__.utils.anchors
+            .getSections('scrollyteller')
+            .map(section => {
+                // Create a node that we can mount onto
+                section.mountNode = document.createElement('div');
+                section.mountNode.className = 'u-full';
+                section.startNode.parentNode.insertBefore(
+                    section.mountNode,
+                    section.startNode
+                );
 
-        // TODO: Work out how to keep a cache of the old DOM so hot reload works nicely
-        // Remove current article from the DOM
-        // if (nextNode) {
-        //     nextNode.previousSibling.remove();
-        // }
+                // Load the config and find any waypoints
+                section.config = alternatingCaseToObject(section.configSC);
+                section.markers = initMarkers(section, 'mark');
+
```

Comment:
I think we should probably also remove the anchors from the page here so they aren't left lying around the DOM.